terragrunt/modules/crates-io/iam.tf

```diff
@@ -0,0 +1,121 @@
+// This file configures the IAM users, policies and roles for crates.io
+
+resource "aws_iam_user" "heroku" {
+  name = "${var.iam_prefix}--heroku"
```

Comment:
Do we need the prefix as a variable? Can we just turn the domain name into the prefix like we do with the `aws_cloudfront_response_headers_policy`?

Reply:
Not sure if IAM supports dots.

Reply:
I don't mean using the domain itself but rather using the domain as the input that gets transformed into something using `_`.